test/test_functional.py

```diff
@@ -33,25 +30,12 @@ def test_clamp(self):
         b_coeffs = torch.tensor([1, 0], dtype=self.dtype, device=self.device)
         a_coeffs = torch.tensor([1, -0.95], dtype=self.dtype, device=self.device)
         output_signal = F.lfilter(input_signal, a_coeffs, b_coeffs, clamp=True)
-        self.assertTrue(output_signal.max() <= 1)
+        assert output_signal.max() <= 1
```

Comment:
nit: changing `self.assertTrue` to `assert`? oh that's because the class isn't inheriting from `unittest`. I see.